gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -321,13 +320,17 @@ private void addMapPolyline(MapPolygon mapPolygon) {
   }
 
   private boolean isPolygonCompleted(List<Point> vertices) {
-    return vertices.size() != stream(vertices).distinct().count();
+    return vertices.get(vertices.size() - 1) == vertices.get(0);
```

Comment:
This will force quit if `vertices` is empty.

Reply:
added check if vertices size is less than 3 return false

Reply:
-> `return vertices.size() > 2 && vertices.get(vertices.size() - 1) == vertices.get(0)`

Also, as always please be sure to run `google-java-format` on code before submitting.